powerline/bindings/lemonbar/powerline-lemonbar.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import time
+import re
+import subprocess
+
+from threading import Lock, Timer
+from argparse import ArgumentParser, REMAINDER
+
+from powerline.lemonbar import LemonbarPowerline
+from powerline.lib.shell import run_cmd
+
+
+if __name__ == '__main__':
+	parser = ArgumentParser(
+		description='Powerline BAR bindings.',
+		usage='%(prog)s [-h] [--i3] [--height HEIGHT] [--interval INTERVAL] [--bar-command CMD] -- args')
```

Comment:
it doesn't really display the `-- args` part well, it looks like this:

    usage: powerline-lemonbar.py [-h] [--i3] [--height HEIGHT]
                                 [--interval INTERVAL] [--bar-command BAR_COMMAND]
                                 ...

    Powerline BAR bindings.

    positional arguments:
      args                  Extra arguments for lemonbar.

but the `--` is mandatory and it doesn't show that.